static/js/containers/DashboardPage.js

```diff
@@ -107,34 +114,50 @@ class DashboardPage extends React.Component {
     dispatch(startCalculatorEdit(currentProgramEnrollment.id));
   };
 
+  setDocumentDate = newDate => {
```

Comment:
this is keeping track of the date that documents were submitted, correct? if so, i think all permutations of 'document date' should change to 'document submit date' or 'document sent date'. 'document date' is pretty ambiguous